linopy/solvers.py

```diff
@@ -471,6 +471,12 @@ def get_solver_solution() -> Solution:
 
             sol = df[variables_b][2]
             dual = df[~variables_b][3]
+
+            # import ipdb
+            # ipdb.set_trace()
+            # TODO the problem is that this sol file has both rows and columns in it, and uses original names
+            # linopy requires highs for MPS files anyway, so we can use highs for MPS to tell which ones are vars
+
```

Comment:
remove this comment?

Reply:
Yes sorry, got added by mistake, will push a fix shortly!